playbooks/robusta_playbooks/grafana_enrichment.py

```diff
@@ -101,11 +103,7 @@ def report_image_changes(event: KubernetesAnyChangeEvent):
         title=f"{FindingSubjectType.TYPE_DEPLOYMENT.value} {event.obj.metadata.name} updated in namespace {event.obj.metadata.namespace}",
         source=FindingSource.KUBERNETES_API_SERVER,
         aggregation_key="report_image_changes",
-        subject=FindingSubject(
-            event.obj.metadata.name,
-            FindingSubjectType.TYPE_DEPLOYMENT,
-            event.obj.metadata.namespace,
-        ),
+        subject=KubeObjFindingSubject(event.obj, FindingSubjectType.TYPE_DEPLOYMENT),
```

Comment:
I think image changes aren't related to a Node. I would just leave the node_name empty here.
In addition, I think I see some old bug here.
It's not always a deployment, it can be any k8s resource.
I think the SubjectType should be taken from the event, and not always be DEPLOYMENT